spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -217,35 +218,22 @@
     describe 'IPP enabled' do
       describe 'DAV not enabled' do
         let!(:pending_enrollments) do
-          [
+          ['BALTIMORE', 'FRIENDSHIP', 'WASHINGTON', 'ARLINGTON', 'DEANWOOD'].map do |name|
             create(
-              :in_person_enrollment, :pending,
-              selected_location_details: { name: 'BALTIMORE' },
-              issuer: 'http://localhost:3000'
-            ),
-            create(
-              :in_person_enrollment, :pending,
-              selected_location_details: { name: 'FRIENDSHIP' }
-            ),
-            create(
-              :in_person_enrollment, :pending,
-              selected_location_details: { name: 'WASHINGTON' }
-            ),
-            create(
-              :in_person_enrollment, :pending,
-              selected_location_details: { name: 'ARLINGTON' }
-            ),
-            create(
-              :in_person_enrollment, :pending,
-              selected_location_details: { name: 'DEANWOOD' }
-            ),
-          ]
+              :in_person_enrollment,
+              :pending,
+              :with_notification_phone_configuration,
+              issuer: 'http://localhost:3000',
+              selected_location_details: { name: name },
+            )
+          end
         end
-        let(:pending_enrollment) { pending_enrollments[0] }
+        let(:pending_enrollment) { pending_enrollments.first }
 
         before do
+          enrollment_record = InPersonEnrollment.where(id: pending_enrollments.map(&:id))
```

Comment:
small nit, just to indicate it's an array and not a single record

```suggestion
          enrollment_records = InPersonEnrollment.where(id: pending_enrollments.map(&:id))
```

Reply:
Thanks! Good catch, I had it as enrollment_records everywhere else, except this test.